pkg/R/DataFrame.R

```diff
@@ -171,6 +171,15 @@ setMethod("showDF",
             cat(callJMethod(x@sdf, "showString", numToInt(numRows)))
           })
 
+setMethod("show", "DataFrame",
+          function(object) {
+            cols <- lapply(dtypes(object), function(l) {
+              paste(l, collapse = ":")
+            })
+            s <- paste(cols, collapse = ", ")
+            cat(paste("DataFrame[", s, "]\n"))
```

Comment:
minor nit: you probably want `sep=""` here. Right now it looks like 
```
DataFrame[ cust_id:string, date:string, day_num:bigint, extended_price:double, sku:bigint, unit_price:double, units:bigint ]
```
so there is an extra space at the beginning and end